package.json

```diff
@@ -28,6 +28,10 @@
     "history": "^4.10.1",
     "react": "16.11.0",
     "react-dom": "16.11.0",
-    "timeago.js": "^4.0.0"
+    "react-redux": "^7.1.3",
```

Comment:
You added these redux dependencies here, but I'm not seeing anywhere in the code below where you use them. Can we take them out?